app/daemon.go

```diff
@@ -81,6 +82,10 @@ func DaemonCmd() cli.Command {
 				Name:  FlagKubeConfig,
 				Usage: "Specify path to kube config (optional)",
 			},
+			cli.BoolFlag{
+				Name:  FlagUpgradeVersionCheck,
+				Usage: "Have a warning log and an event for unrecommended upgrade path",
```

Comment:
Usage should be `Enforce version checking for upgrades. If disabled, there will be no requirement for the necessary upgrade source version`

Reply:
Modified. Thanks